src/clr-angular/forms/index.ts

```diff
@@ -4,3 +4,4 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 export * from "./common/index";
+export * from "./datepicker/index";
```

Comment:
@youdz @gnomeontherun: Marking this so that I can make sure you review this.

Reply:
This is correct.